addons/website_sale/static/tests/tours/website_sale_shop_mail.js

```diff
@@ -84,18 +84,6 @@ tour.register('shop_mail', {
     {
         content: "wait mail to be sent, and go see it",
         trigger: '.o_Message_content:contains("Your"):contains("order")',
-        run: function () {
-            window.location.href = "/web#action=mail.action_view_mail_mail&view_type=list";
-        },
-    },
-    {
-        content: "click on the first email",
-        trigger: '.o_data_cell:contains("(Ref S")',
-    },
-    {
-        content: "check it's the correct email, and the URL is correct too",
-        trigger: 'div.o_field_html[name="body_content"] p:contains("Your"):contains("order")',
-        extra_trigger: 'div.o_field_html[name="body_content"] a[href^="https://my-test-domain.com"]',
```

Comment:
This whole thing is moved to python since we don't actually need to check the technical email view is there